runtime/kusama/src/lib.rs

```diff
@@ -1513,10 +1513,31 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
+	SetStakingLimits,
 >;
 /// The payload being signed in the transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+pub struct SetStakingLimits;
+impl frame_support::traits::OnRuntimeUpgrade for SetStakingLimits {
+	fn on_runtime_upgrade() -> Weight {
+		// This will be the threshold needed henceforth to become a nominator. All nominators will
+		// less than this amount bonded are at the risk of being chilled by another reporter.
+		let min_nominator_bond = UNITS / 1000;
```

Comment:
While the polkadot limit is more educated, for this one I just pulled `KSM / 1000` out of thin air as a relatively lenient minimum.

Reply:
```suggestion
		let min_nominator_bond = UNITS / 10;
```